### PR TITLE
feat: document more error codes for Accelerate

### PR DESCRIPTION
**Pull request by ankur-arch**

### CODE REVIEW

File: content/400-reference/200-api-reference/250-error-reference.mdx [hunk 1]
```diff
@@ -525,6 +525,22 @@ Interactive transaction error:
 
 Accelerate-related errors start with `P6xxx`.
 
+#### <inlinecode>P6000</inlinecode> (<inlinecode>ServerError</inlinecode>)
+
+Generic Error to catch all other errors.
```
Comment (nikolasburk) [inline content/400-reference/200-api-reference/250-error-reference.mdx]:
```suggestion
Generic error to catch all other errors.
```

File: content/400-reference/200-api-reference/250-error-reference.mdx [hunk 2]
```diff
@@ -525,6 +525,22 @@ Interactive transaction error:
...
+
+The URL is malformed; for instance, it does not use the `prisma://` protocol.
+
+#### <inlinecode>P6002</inlinecode> (<inlinecode>Unauthorized</inlinecode>)
+
+The API Key in the connection string is invalid.
+
+#### <inlinecode>P6003</inlinecode> (<inlinecode>PlanLimitReached</inlinecode>)
+
+The included usage of the current plan has been exceeded. This can only occur on the free plan.
```
Comment (nikolasburk) [inline content/400-reference/200-api-reference/250-error-reference.mdx]: maybe link "free plan" to our pricing page?

File: content/400-reference/200-api-reference/250-error-reference.mdx [hunk 3]
```diff
@@ -533,6 +549,14 @@ The global timeout of Accelerate has been exceeded. You can find the limit [here
 
 The user supplied invalid parameters. Currently only relevant for transaction methods. For example, setting a timeout that is too high. You can find the limit [here](/data-platform/accelerate/limitations#interactive-transactions-query-timeout-limit).
 
+#### <inlinecode>P6006</inlinecode> (<inlinecode>VersionNotSupported</inlinecode>)
+
+The chosen Prisma version is not compatible with Accelerate. This may occur when a user uses an unstable development version that we occasionally prune.
+
+#### <inlinecode>P6008</inlinecode> (<inlinecode>ConnectionError|EngineStartError</inlinecode>)
+
+Engine did not start. E.g. could not connect to database.
```
Comment (nikolasburk) [inline content/400-reference/200-api-reference/250-error-reference.mdx]: What does the "E.g. could not connect to database." mean exactly? Maybe we can use a few more words to elaborate and provide context?

**Review by nikolasburk — CHANGES_REQUESTED** [review]
three minor suggestions but otherwise this looks good to me 👍

File: content/400-reference/200-api-reference/250-error-reference.mdx (hunk 2 above)
Comment (ankur-arch) [reply content/400-reference/200-api-reference/250-error-reference.mdx]: That's a good idea, I'll do that

**Review by nikolasburk — APPROVED** [review]
lgtm 👍

**Pull request merged**